src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -1348,7 +1348,7 @@ pub const MINCORE_SUPER: ::c_int = 0x20;
 
 const_fn! {
     {const} fn _CMSG_ALIGN(n: usize) -> usize {
-        (n + 3) & !3
+        (n + 7) & !7
```

Comment:
Better forwards-compatibility with any 32-bit builds that Dragonfly might add in the future.
```suggestion
        (n + ::mem::size_of::<::c_long>()) & !::mem::size_of::<::c_long>()
```

Reply:
Whoops, we dropped the subtraction by one here (and I didn't catch it either).

Fix: https://github.com/rust-lang/libc/pull/2644